recipe/meta.yaml

```diff
@@ -16,10 +16,10 @@ source:
     - remove_linker_flags.patch
     - fix_ninja_build.patch  # [win]
     - remove_rpaths.patch  # [linux]
+    - fix_tests.patch
 
 build:
   number: {{ build }}
-  skip: True  # [TTK_WITH_PARAVIEW and linux and py==36]
   skip: True  # [TTK_WITH_PARAVIEW and osx and py==37]
   skip: True  # [TTK_WITH_PARAVIEW and win and py==39]
```

Comment:
Please change to `skip: True  # [TTK_WITH_PARAVIEW and win]` for now, since this times out.